charts/aws-mountpoint-s3-csi-driver/templates/node.yaml

```diff
@@ -91,6 +91,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: HOST_TOKEN_DIR
```

Comment:
Thanks for catching this. My preference would actually be to eliminate the use of `HOST_TOKEN_DIR` in the code in favor of just reusing `HOST_PLUGIN_DIR` given they're set to the same thing. If you're not set up to make changes to the go code, we can fix this before the next release. Of course we're happy to take the contribution with an updated pr.

Reply:
Yeah, it's effectively a better approach.  I will update the PR with the modified code.
Thanks